<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>استبدال النقاط بهديه</string>
  </property>
  <widget class="QPushButton" name="Qbtn_search">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>29LT Bukra Regular</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>بحث</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Qbtn_replace">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>29LT Bukra Regular</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
</string>
   </property>
   <property name="text">
    <string>إستبدال</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>111</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>29LT Bukra Regular</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>29LT Bukra Regular</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="Qline_name">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>رقم العميل</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>اسم  العميل</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Qline_cust">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
    </property>
   </widget>
   <zorder>label_15</zorder>
   <zorder>Qline_name</zorder>
   <zorder>label_14</zorder>
   <zorder>Qline_cust</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>29LT Bukra Regular</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="CMB_redeemItem">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>179</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="Qline_points">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>نقاط  العميل</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>اختر الهديه</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>29LT Bukra Regular</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>القيمه المستحقه</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>عدد النقاط المستبدله</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>النقاط  المتبقيه</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Qline_replace">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>10</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Qline_remainder">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="Qline_point_value">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>82</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>29LT Bukra Regular</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QLineEdit:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                       QLineEdit:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>Qbtn_search</zorder>
  <zorder>Qbtn_replace</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
